cmake/EthCompilerSettings.cmake

```diff
@@ -33,10 +33,10 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 	add_compile_options(-Werror)
 
 	# Configuration-specific compiler settings.
-	set(CMAKE_CXX_FLAGS_DEBUG          "-O0 -g -DETH_DEBUG")
+	set(CMAKE_CXX_FLAGS_DEBUG          "-O0 -g3 -DETH_DEBUG")
 	set(CMAKE_CXX_FLAGS_MINSIZEREL     "-Os -DNDEBUG")
 	set(CMAKE_CXX_FLAGS_RELEASE        "-O3 -DNDEBUG")
-	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g")
+	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g3")
 
```

Comment:
Are these changes in the cmake file required to limit `check_style.sh`?

Reply:
No this is an independent change. I can separate this into a separate pull request if desird.

Reply:
Please do not sneak in such changes. Also, please explain why it does and why you think it is useful.

Reply:
Honestly I wasn't to be sneaky, and take offense at the suggestion. I haded hoped to speed up PR reviews, with what I had hoped would be non-controversial small (2-line) changes. 

That said, I see the error of my ways and I see that this is slowing things down. So I will put in the -g -> -g3 change in a separate PR. 

Please understand I am still learning the ropes here. Things that may be obvious to others on the project are not to me.

Reply:
Oh I'm sorry, this was not meant to be offensive - most of us are not native speakers, so I hope you excuse us. The thing is that compiler flag changes might result in very drastic results. Yes, I think each item (as tiny as it might be) in a separate PR is a faster approach since the PRs can then be merged independently.

Reply:
> most of us are not native speakers,

You have my utmost respect here. Your English is far far far better than my ability to understand any language other then English.  

Another PR for -g3 is in, but I see you've already commented on that.